app/modules/parsing/tasks/cache_cleanup_tasks.py

```
@@ -0,0 +1,62 @@
+    try:
+        db = next(get_db())
+        cleanup_service = CacheCleanupService(db)
+
+        deleted_count = cleanup_service.cleanup_least_accessed(max_entries)
+
+        if deleted_count > 0:
+            logger.info(
+                f"Cache size cleanup completed: {deleted_count} entries removed"
+            )
+
+        return {"deleted_count": deleted_count, "status": "success"}
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)